### DISCUSSION TITLE
next/image with remote URLs and caching?

### DISCUSSION THREAD

**Discussion by clam61** [vercel/next.js#48368] [Help]
### Summary

I am trying to optimize a website.  We just went to production and our home page loads a few hundred very small images from our CMS.  We load these through a next/image tag so of course the URL is remote and so the result is something like _next/image?url=https://...

My question is if these images are cached statically somehow in the nextjs static folder, and if so, how long is the cache and how can I change the cache time?  If it's not cached then each page load is 200+ requests to our CDN.  This is opposed to storing them statically in Vercel which may or may not result in lower costs.

**Comment by MauriceArikoglu** [comment]
Wondering about the same thing. @clam61 did you find out more?
